### DISCUSSION TITLE
Consider adding wheels to releases

### DISCUSSION THREAD

**Discussion by tusharsadhwani** [Textualize/textual#99] [Ideas]
I believe dropping a pre-built `.whl` file into the releases would go a long way.

If going the manual route doesn't sound that nice, there are github actions that will build and release every new tag for you.

**Comment by ubaumann** [comment]
You can just do something like

`poetry add git+https://github.com/willmcgugan/textual.git`

Works also with pip

Or 

`poetry add git+https://github.com/willmcgugan/textual.git#v0.1.11`

**Comment by willmcgugan** [comment]
@tusharsadhwani How would you use those .whl files? Does @ubaumann 's suggestion work for you?

**Reply by tusharsadhwani** [reply]
It does work, yeah.

Wheels are nice because they can be installed by doing `pip install x.whl`, kindof like single-file installers, in case pypi stops working or whatever.

But since installing from git works, that's a non issue.

**Reply by willmcgugan** [reply]
If PyPi stops working, it will be the end of the world and we will be fighting zombies rather than building TUIs.
